lua/doom/modules/features/indentlines/init.lua

```diff
@@ -1,23 +1,24 @@
 local indentlines = {}
 
 indentlines.settings = {
-  char = "│",
-  use_treesitter = true,
-  show_first_indent_level = false,
```

Comment:
Removing the first indent level is not an option anymore but it can be done using hooks:

```lua
  local hooks = require("ibl.hooks")
  hooks.register(
    hooks.type.WHITESPACE,
    hooks.builtin.hide_first_space_indent_level
  )
```

Reply:
Maybe we could leave it like this? I don't think having the first indent level is annoying or affects performance. To be honest, I had deleted it because I didn't like it, but this isn't a personal setup and the plugin configurations can be easily modified anyway.